server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java

```diff
@@ -752,7 +752,7 @@ private CompletionStage<RestResponse> getCacheConfig(RestRequest request) {
    private CompletionStage<RestResponse> getCacheAvailability(RestRequest request) {
       String cacheName = request.variables().get("cacheName");
       // Use EmbeddedCacheManager directly to allow internal caches to be updated
-      if (!invocationHelper.getRestCacheManager().getInstance().isRunning(cacheName))
+      if (!invocationHelper.getRestCacheManager().getInstance().cacheExists(cacheName))
```

Comment:
This and below can't be like this. The isRunning call is needed to protect the getCache method.

Reply:
Sorry, I missed this one :/ Updated.